gulp/tasks/test_ci.js

```diff
@@ -0,0 +1,14 @@
+(function () {
+    'use strict';
+
+    var gulp = require('gulp'),
+        karma = require('karma').server,
+        path = require('path'),
+        configFile = 'karma.ci.conf.js';
+
+    gulp.task('test_ci', function (callback) {
```

Comment:
I agree @andy-armstrong, I think Travis should run the same tests as local unless there is a good reason for them to differ that I'm unaware of. It would make debugging more difficult too.